src/java/org/apache/cassandra/db/commitlog/SimpleCachedBufferPool.java

```diff
@@ -37,6 +37,8 @@
 public class SimpleCachedBufferPool
 {
     private static final EnumMap<BufferType, FastThreadLocal<ByteBuffer>> reusableBBHolder = new EnumMap<>(BufferType.class);
+    // Convenience variable holding a ref to the current resuableBB to avoid map lookups
+    private static FastThreadLocal<ByteBuffer> reusableBB;
```

Comment:
This should not be static.

Reply:
Ok I see it now. That BB can be of any BBType and as we no longer switch on the fly between types it can't be static.